### DISCUSSION TITLE
Exercism learning exercise support

### DISCUSSION THREAD

**Discussion by Ifycode** [Ifycode-support/discussions#1] [Exercism Issues]
What specific issues are you having with exercise 3? @Danbaba1

**Comment by Ifycode** [comment]
You know what? Using github discussions maybe overwhelming and a bit hard to track. So I've created a repo instead to make it easier.

Do the following:
- Go to this repo: https://github.com/Ifycode-support/Danbaba1-exercism-issues
- Click on the **issues** tab. Then create a new issue for this learning exercise 3. Be specific about the issue you are having in there.
- Watch this video to learn about how to fork a repo, how to create a new branch from the **main** branch, how to make changes in your new branch and how to make and send a pull request. Those are the only things you need from that video. Ignore any other thing. Here's the link to the video: https://obiagba-mary.gitbook.io/gads-projects/contribution-guide/video-tutorials
- There's a file in the github repo called 3.js. Now create a new branch from main, replace the content of the 3.js file with your solution and then send a pull request.

Let me know if you don't understand any of what i've written so far @Danbaba1

**Reply by Danbaba1** [reply]
okay ill try it out
